crates/api_models/src/payments.rs

```diff
@@ -638,6 +646,8 @@ pub enum MandateType {
     SingleUse(MandateAmountData),
     /// If the mandate should be valid for multiple debits
     MultiUse(Option<MandateAmountData>),
+
+    UpdateMandateId(String),
```

Comment:
Nit: We're updating the mandate, not the ID.
```suggestion
    UpdateMandate(String),
```